testgrid/config/config.yaml

```diff
@@ -1906,82 +1890,24 @@ dashboards:
-    base_options: 'include-filter-by-regex=Kubectl%7Ckubectl'
-    description: 'kubectl e2e tests in gce-1.6-1-7-gci-kubectl-skew'
-  - name: gke-1.6-1-7-gci-kubectl-skew
-    test_group_name: ci-kubernetes-e2e-gke-1-6-1-7-gci-kubectl-skew
-    base_options: 'include-filter-by-regex=Kubectl%7Ckubectl'
-    description: 'kubectl e2e tests in gke-1.6-1-7-gci-kubectl-skew'
-  - name: gce-1.6-1-7-gci-kubectl-skew-serial
-    test_group_name: ci-kubernetes-e2e-gce-1-6-1-7-gci-kubectl-skew-serial
-    base_options: 'include-filter-by-regex=Kubectl%7Ckubectl'
-    description: 'kubectl e2e tests in gce-1.6-1-7-gci-kubectl-skew-serial'
-  - name: gke-1.6-1-7-gci-kubectl-skew-serial
-    test_group_name: ci-kubernetes-e2e-gke-1-6-1-7-gci-kubectl-skew-serial
-    base_options: 'include-filter-by-regex=Kubectl%7Ckubectl'
-    description: 'kubectl e2e tests in gke-1.6-1-7-gci-kubectl-skew-serial'
-  - name: gce-1-7-1.6-cvm-kubectl-skew
-    test_group_name: ci-kubernetes-e2e-gce-1-7-1-6-cvm-kubectl-skew
-    base_options: 'include-filter-by-regex=Kubectl%7Ckubectl'
-    description: 'kubectl e2e tests in gce-1-7-1.6-cvm-kubectl-skew'
-  - name: gke-1-7-1.6-cvm-kubectl-skew
-    test_group_name: ci-kubernetes-e2e-gke-1-7-1-6-cvm-kubectl-skew
-    base_options: 'include-filter-by-regex=Kubectl%7Ckubectl'
-    description: 'kubectl e2e tests in gke-1-7-1.6-cvm-kubectl-skew'
-  - name: gce-1-7-1.6-cvm-kubectl-skew-serial
-    test_group_name: ci-kubernetes-e2e-gce-1-7-1-6-cvm-kubectl-skew-serial
-    base_options: 'include-filter-by-regex=Kubectl%7Ckubectl'
-    description: 'kubectl e2e tests in gce-1-7-1.6-cvm-kubectl-skew-serial'
-  - name: gke-1-7-1.6-cvm-kubectl-skew-serial
-    test_group_name: ci-kubernetes-e2e-gke-1-7-1-6-cvm-kubectl-skew-serial
-    base_options: 'include-filter-by-regex=Kubectl%7Ckubectl'
-    description: 'kubectl e2e tests in gke-1-7-1.6-cvm-kubectl-skew-serial'
-  - name: gce-1-7-1.6-gci-kubectl-skew
-    test_group_name: ci-kubernetes-e2e-gce-1-7-1-6-gci-kubectl-skew
-    base_options: 'include-filter-by-regex=Kubectl%7Ckubectl'
-    description: 'kubectl e2e tests in gce-1-7-1.6-gci-kubectl-skew'
-  - name: gke-1-7-1.6-gci-kubectl-skew
-    test_group_name: ci-kubernetes-e2e-gke-1-7-1-6-gci-kubectl-skew
-    base_options: 'include-filter-by-regex=Kubectl%7Ckubectl'
-    description: 'kubectl e2e tests in gke-1-7-1.6-gci-kubectl-skew'
-  - name: gce-1-7-1.6-gci-kubectl-skew-serial
-    test_group_name: ci-kubernetes-e2e-gce-1-7-1-6-gci-kubectl-skew-serial
-    base_options: 'include-filter-by-regex=Kubectl%7Ckubectl'
-    description: 'kubectl e2e tests in gce-1-7-1.6-gci-kubectl-skew-serial'
-  - name: gke-1-7-1.6-gci-kubectl-skew-serial
-    test_group_name: ci-kubernetes-e2e-gke-1-7-1-6-gci-kubectl-skew-serial
-    base_options: 'include-filter-by-regex=Kubectl%7Ckubectl'
-    description: 'kubectl e2e tests in gke-1-7-1.6-gci-kubectl-skew-serial'
+  - name: gce-last_release-master-cvm-kubectl-skew
```

Comment:
probably worth explicitly say this is `gce-1.7-master-`, it's just a display name and easy to change. Not have too strong preference.